Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/ReplaceOwnerFromInviteFlow.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ReplaceOwnerFromInviteFlow.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/16/22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+class ReplaceOwnerFromInviteFlow: ReplaceOwnerFlow {
```

Comment:
Can you please add a comment with the intended steps for this and other flows? You can see example in the other flows.

This way it is documented what screen sequence is implemented, and when you read the code you don't need to jump around the class hierarchy